src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public enum Colors {

```

Comment:
```suggestion
public enum Color {
```
I would recommend to use singular for enum names